BrainPortal/lib/scir_pbs.rb

```diff
@@ -31,8 +31,20 @@ class ScirPbs < Scir
   class Session < Scir::Session #:nodoc:
 
     def update_job_info_cache #:nodoc:
-      out, err = bash_this_and_capture_out_err("qstat -f")
-      raise "Cannot get output of 'qstat -f' ?!?" if out.blank? && ! err.blank?
+      qstat_command = "qstat -u #{CBRAIN::Rails_UserName.to_s.bash_escape} -f"
+      retry_count = 0
+      # In some cases, the standard output of the qstat command 
+      # is blank even though jobs are still running on the cluster. In this situation,
+      # all the active jobs will be marked completed while they are not. 
+      # To address this issue, we retry the qstat command up to 3 times with a 3-second interval
+      # as long as the output of qstat is blank.
+      while(retry_count < 3) do
```

Comment:
This is a bit costly... but OK. It would be great if we could mark individual bourreaux as 'unreliable' and only those would retry; we'll do it in a future version.